pkg/util/pki/subject_test.go

```diff
@@ -192,9 +181,10 @@ func FuzzRoundTripRDNSequence(f *testing.F) {
 		}
 		for _, rdn := range rdnSeq {
 			for _, tv := range rdn {
-				// Skip if the String() function will return a literal OID type, as we
-				// do not yet support parsing these.
-				if _, ok := knownMarshalTypes[tv.Type.String()]; !ok {
+				// Skip if the Type was not recognized. The String() output will be
+				// an invalid type, value pair with empty type, which will give a "DN ended with
+				// an incomplete type, value pair" error when parsing.
+				if tv.Type.String() == "" {
 					t.Skip()
 				}
```

Comment:
I can't tell whether this fuzzing function is now testing subject strings with literal OIDs.

I expected to see a new fixed sample at the start of this fuzzing function, maybe matching the example in the release note of the PR.
```go
f.Add("1.2.3.4=String Value")
```

Reply:
Or use the example from the RFC:

> In this RDN example, the type in the RDN is unrecognized, and the
> value is the BER encoding of an OCTET STRING containing two octets,
> 0x48 and 0x69.
>    1.3.6.1.4.1.1466.0=#04024869
-- https://www.rfc-editor.org/rfc/rfc4514#section-4



Zeilenga                    Standards Track                     [Page 7]

[RFC 4514](https://www.rfc-editor.org/rfc/rfc4514)               LDAP: Distinguished Names               June 2006


      1.3.6.1.4.1.1466.0=#04024869

Reply:
I added both examples.